include/envoy/secret/secret_manager.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <google/protobuf/util/json_util.h>
+
+#include <iomanip>
```

Comment:
not needed here and next line. In general please trim headers to what is needed. Comment applies elsewhere.

Reply:
Removed